android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldRadioView.java

```diff
@@ -47,6 +47,7 @@
 
     protected View parentView;
     private TextView hintTextView;
+    private TextView labelTextView;
```

Comment:
Did you confirm that it's working with old themes? Maybe it's better to add a null check thinking in customers that may have custom layouts using DDLFields.

Reply:
Hmm, in this case, I just added as a class property because the view was referenced in several places with findViewById. Every field has labels, even in the custom themes.

Reply:
But to ensure that it's working fine with old themes we have to mock different field types or wait for the new APIs integration.